### ISSUE TITLE
RegionDs9/ds9parse.y requires bison 3.0

### ISSUE THREAD

**Issue by vsuorant** [Astroua/CARTAvis#174]
RegionDs9/ds9parse.y requires bison 3.0, but Red Hat 6 comes with Bison 2.4.1 and Red Hat 7 with 2.7 . Getting version 3 to compile on these platforms isn't trivial (at least auto-tools would need to be rebuilt also). Could ds9parse.y be refactored or downgraded to support 2.4.1?

**Comment by slovelan** [comment]
Once my branch is merged into develop, ds9parse.y should be downgraded to support 2.4.1.
